src/overlays/actors/ovl_En_Wf/z_en_wf.c

```diff
@@ -1181,7 +1181,7 @@ void func_80992E0C(EnWf* this, PlayState* play) {
         } else {
             Item_DropCollectibleRandom(play, &this->actor, &this->actor.world.pos, 0x60);
         }
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```

Comment:
Can replace with an early return.

Reply:
Does not match:
![image](https://user-images.githubusercontent.com/12245827/195777628-61f99ad3-2bc8-4311-9335-6318625385b5.png)